falcon/vendor/mimeparse/__init__.py

```diff
@@ -1,6 +1,7 @@
 """
-This module wraps code from the MIT-licensed python-mimeparse project. The
-original project source code may be obtained from GitHub by visiting the
+This module wraps code from the MIT-licensed python-mimeparse project.
+
+Theoriginal project source code may be obtained from GitHub by visiting the
```

Comment:
Word rewrapping is a bit off here, _Theoriginal_ (sic) should not be merged.

Reply:
Unsure if you wanted a specific change for the word wrapping. I did update `Theorigin..` > `The original`